tedx-website/src/components/landing.css

```diff
@@ -0,0 +1,119 @@
+
+  html {
+    /* set background color */
+    background-color: black;
+  }
+  body{
+    background-color: transparent !important;
+    max-width: 100%;
+    overflow-x: hidden !important;
+  }
+  .grid{
+    margin-left: 10vh;   
+    margin-right: 10vh;
+  }
+  
+  .title{
+    font-size: 5vw;
```

Comment:
I'd do something like `calc(2.3rem + 1.5vw)` instead to make it larger than the normal `h1` (I just took this calculation from the bootstrap h1 font-size and increased it a bit haha). That way you don't need the media-queries for `.title` either on line 84 and 92.